contributors/distribution-guidelines.md

```diff
@@ -1,17 +1,42 @@
+Additionally, commands used to execute the distribution must be different from those in Chef Software's distribution. For example, if you create a distribution of Chef, the command to use your distribution's binary cannot be "chef", though it can be something like "kitty" - the goal is prevent confusion about which distributions come from Chef Software and which come from another source.
+
+## What are some examples?
+
+The Chef community has begun the process of creating a Community distribution of Chef itself - beginning with implementing a way to handle Chef’s trademarks throughout the code. They created a Chef::Dist namespace and replaced all Chef trademarks in the code with a configurable variable - i.e. “Chef Server” became “#{Chef::Dist::PRODUCT} Server”.
+
+This pull request was created in collaboration and reviewed by Chef Software and is a good implementation of what we are looking for when requiring that distros remove Chef’s trademarks. Please feel free to use it as a reference: https://github.com/chef/chef/pull/8368
+
+You can also see the conversation around the decisions of that pull request in this design proposal: https://github.com/chef/chef/issues/8376
+
+## What about Chef code libraries used in distributions?
+
+You do not have to remove Chef from the existing Ruby class names, e.g. Chef::Log. (However, derived or modified class names should not include Chef).
+
+## Do you need to change the installation paths for distributions?
+
+Yes, do not reuse /opt/chef, /etc/chef, etc. as installation paths. It is fine if the distribution uses symlinks or other tools to make migration easier.
+
+## Can you reference the Chef gem?
+
+Yes. Chef is currently removing binstubs from the Chef gem to allow it to function as a library, rather than an executable. It is fine to reference the Chef gem as a library from your code, but derived and modified class names should not include Chef trademarks.
 
-You do not have to remove Chef from the existing Ruby class names, e.g. Chef::Log.  (Derived or modified class names should not include Chef).
+## Do you need to change config variables? 
 
-If you have specific questions, please reach out to us, we would love to talk to you!
\ No newline at end of file
+Yes. Because it is something external to the software that a user interacts with. Again, it will create some breakage but the variable prefix shouldn't misrepresent the actual distribution being run. 
+Will these guidelines ever change?
+That is possible. Again, the purpose of the Chef Trademark Policy is to avoid confusion in the market about which distributions come from Chef and which distributions come from other sources. If we find that these guidelines are not sufficient to avoid that confusion, we may need to revisit them. We will, of course, give ample notice of any changes to these guidelines to allow for plenty of time to implement them.
```

Comment:
I'd go with "yes, they will", since there's a few statements that say "we're in the process of..." or "this or that is being modified", implying fluidity. Might as well make it explicit no?

Reply:
Fixed!